package.json

```diff
@@ -119,19 +118,24 @@
     "prettier": "2.4.1",
     "resolve": "1.22.10",
     "rimraf": "3.0.2",
-    "rollup": "4.34.9",
-    "rollup-plugin-babel": "4.4.0",
-    "rollup-plugin-commonjs": "10.1.0",
-    "rollup-plugin-includepaths": "0.2.4",
-    "rollup-plugin-json": "4.0.0",
-    "rollup-plugin-node-globals": "1.4.0",
-    "rollup-plugin-node-resolve": "5.2.0",
-    "rollup-plugin-replace": "2.2.0",
-    "rollup-plugin-uglify": "6.0.4",
-    "rollup-watch": "4.3.1",
     "streamtest": "2.0.0",
     "ts-jest": "26.5.6",
     "tsconfig-replace-paths": "0.0.14",
     "typescript": "4.5.2"
+  },
+  "preconstruct": {
+    "packages": [
+      "packages/api-request-builder",
+      "packages/http-user-agent",
+      "packages/sdk-auth",
+      "packages/sdk-client",
+      "packages/sdk-middleware-auth",
+      "packages/sdk-middleware-correlation-id",
+      "packages/sdk-middleware-http",
+      "packages/sdk-middleware-logger",
+      "packages/sdk-middleware-queue",
+      "packages/sdk-middleware-user-agent",
+      "packages/sync-actions"
```

Comment:
These are all packages which are configured as SDKs and were prior built with Rollup.